web/models/user.ex

```diff
@@ -32,6 +37,15 @@ defmodule CodeCorps.User do
     |> put_pass_hash()
   end
 
+  def update_changeset(struct, params) do
+    struct
+    |> changeset(params)
+    |> cast(params, [:first_name, :last_name, :twitter, :biography, :website])
+    |> validate_format(:website, ~r/\A((http|https):\/\/)?[a-z0-9]+([\-\.]{1}[a-z0-9]+)*\.[a-z]{2,}(([0-9]{1,5})?\/.*)?#=\z/ix)
+    |> validate_format(:twitter, ~r/\A[a-zA-Z0-9_]{1,15}\z/)
+    |> prefix_url(:website)
```

Comment:
I don't think it really matters. All functions will run either way. Validation doesn't break the process, it just adds to the `errors` `Keyword` in the changeset. Preference, really.